<ui version="4.0" >
 <class>LeechCraft</class>
 <widget class="QMainWindow" name="LeechCraft" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>LeechCraft</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mainresources.qrc" >
    <normaloff>:/resources/images/mainapp.png</normaloff>:/resources/images/mainapp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>67</y>
     <width>800</width>
     <height>512</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <item>
     <widget class="QSplitter" name="MainSplitter_" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="PluginsTree_" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="headerHidden" >
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick" >
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QWidget" native="1" name="PluginControls_" />
         </item>
         <item>
          <widget class="QTreeView" name="PluginTasksTree_" />
         </item>
        </layout>
       </widget>
       <widget class="QWidget" native="1" name="PluginAdditionalInfo_" />
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>Actions</string>
    </property>
    <addaction name="ActionAddTask_" />
    <addaction name="separator" />
    <addaction name="separator" />
    <addaction name="ActionQuit_" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>Settings</string>
    </property>
    <addaction name="ActionSettings_" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="ActionAboutLeechCraft_" />
    <addaction name="ActionAboutQt_" />
   </widget>
   <widget class="QMenu" name="PluginsMenu_" >
    <property name="title" >
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="PluginsMenu_" />
   <addaction name="menuSettings" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>579</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="Toolbar_" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>28</y>
     <width>85</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionAddTask_" />
   <addaction name="ActionSettings_" />
  </widget>
  <widget class="QToolBar" name="PluginsToolbar_" >
   <property name="geometry" >
    <rect>
     <x>85</x>
     <y>28</y>
     <width>715</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="ActionAddTask_" >
   <property name="icon" >
    <iconset resource="mainresources.qrc" >
     <normaloff>:/resources/images/main_addjob.png</normaloff>:/resources/images/main_addjob.png</iconset>
   </property>
   <property name="text" >
    <string>Add task</string>
   </property>
   <property name="statusTip" >
    <string>Adds job to a supporting plugin</string>
   </property>
  </action>
  <action name="ActionSettings_" >
   <property name="icon" >
    <iconset resource="mainresources.qrc" >
     <normaloff>:/resources/images/main_preferences.png</normaloff>:/resources/images/main_preferences.png</iconset>
   </property>
   <property name="text" >
    <string>Settings</string>
   </property>
  </action>
  <action name="ActionQuit_" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="ActionAboutLeechCraft_" >
   <property name="text" >
    <string>About LeechCraft...</string>
   </property>
  </action>
  <action name="ActionAboutQt_" >
   <property name="text" >
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainresources.qrc" />
 </resources>
 <connections/>
</ui>
